controllers/admin/AdminSearchConfController.php

```diff
@@ -181,6 +181,49 @@ public function __construct()
                             ),
                         ),
                     ),
+                    'PS_SEARCH_FUZZY' => array(
+                        'title' => $this->trans('Fuzzy Search', array(), 'Admin.Shopparameters.Feature'),
+                        'validation' => 'isBool',
+                        'cast' => 'intval',
+                        'type' => 'bool',
+                        'desc' => $this->trans(
+                                'By default, fuzzy search is enabled and spelling error are allowed. To search “bird”, you can to enter “burd”, “bard”, “beerd” etc...',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ) . '<br/>' .
+                            $this->trans(
+                                'With this option disabled, to get good result, you need get exact writting contained in the word.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                        'hint' => array(
+                            $this->trans(
+                                'Enable spelling errors during search.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                            $this->trans(
+                                'Fuzzy search will check if a close word of the search word exist in indexed words in database. This will be very resource-consuming.',
```

Comment:
Do we need this wording again? It seems we are repeating ourselves since we already explained it in the first tooltip.

Reply:
Done